src/nautilus-window.c

```diff
@@ -527,9 +543,36 @@ const GActionEntry win_entries[] = {
 	{ "prompt-root-location", action_prompt_for_location_root },
 	{ "prompt-home-location", action_prompt_for_location_home },
 	{ "go-to-tab", NULL, "i", "0", action_go_to_tab },
+	{ "custom-key-press", NULL, "s", "''", action_custom_key_press },
```

Comment:
There is no reason why the accelerators should go through a separate action - we already install actions for every script.